doc/howto/images_manage.md

```diff
@@ -0,0 +1,105 @@
+(images-manage)=
+# How to manage images
+
+When working with images, you can inspect various information about the available images, view and edit their properties and configure aliases to refer to specific images.
+You can also export an image to a file, which can be useful to {ref}`copy or import it <images-copy>` on another machine.
+
+## List available images
+
+To list all images on a server, enter the following command:
+
+    lxc image list [<remote>:]
+
+If you do not specify a remote, the {ref}`default remote <images-remote-default>` is used.
+
+(images-manage-filter)=
+### Filter available images
+
+To filter the results that are displayed, specify a part of the alias or fingerprint after the command.
+For example, to show all Debian images, enter the following command:
+
+    lxc image list images: debian
+
+You can specify several filters as well.
+For example, to show all 64-bit Debian images, enter the following command:
+
+    lxc image list images: debian amd64
+
+To filter for properties other than alias or fingerprint, specify the filter in `<key>=<value>` format.
+For example:
+
+    lxc image list images: debian architecture=x86_64
+
+## View image information
+
+To view information about an image, enter the following command:
+
+    lxc image info <image_ID>
+
+As the image ID, you can specify either the image's alias or its fingerprint.
+For a remote image, remember to include the remote server (for example, `images:ubuntu/22.04`).
+
+To display only the image properties, enter the following command:
+
+    lxc image show <image_ID>
+
+You can also display a specific image property (located under the `properties` key) with the following command:
+
+    lxc image get-property <image_ID> <key>
+
+For example, to show the release name of the official Ubuntu 22.04 image, enter the following command:
+
+    lxc image get-property ubuntu:22.04 release
+
+(images-manage-edit)=
+## Edit image properties
+
+To set a specific image property that is located under the `properties` key, enter the following command:
+
+    lxc image set-property <image_ID> <key>
+
+To edit the full image properties, including the top-level properties, enter the following command:
+
+    lxc image edit <image_ID>
```

Comment:
It's kinda awkward that images use the `properties` key for what we call `config` everywhere else, so we need to make a difference between "image properties" and "image properties under the `properties` key".

But I guess there isn't anything we can do about this?

And also, is there any reason for calling it differently for images, or is that just "by accident"?

Reply:
One for @stgraber as I don't know the history on this.

Reply:
It's not an accident, though indeed the naming of `properties` is a bit unfortunate.

We very specifically do not want images to be able to provide configuration to LXD as this would turn into a very easy attack vector. To make this very clear to users and to avoid us accidentally introducing actual configuration as part of what's shipped by an image, we made sure not to have a `config` on images, but just `properties` which don't alter LXD's behavior and is effectively purely cosmetic.

Reply:
Thanks! I added a note to clarify that: "These properties can be used to convey information about the image. They do not configure LXD’s behavior in any way."

Reply:
Sounds good!